### PR TITLE
Create ants-registrationsynquick.json

### PR DESCRIPTION
**Pull request by ltorres6**

### CODE REVIEW

**Review by pvelasco — APPROVED** [review]
Looks good to me!

File: gears/flywheel/ants-registrationsynquick.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  },
+  "license": "Other",
+  "inputs": {
+    "api_key": {
+      "base": "api-key"
+    },
+    "moving_image": {
+      "description": "Image to apply transformation to (generally a coregistered functional). Maps to a command-line argument: %s.",
+      "base": "file",
+      "optional": false,
```
Comment (jiavila) [inline gears/flywheel/ants-registrationsynquick.json:28]: i seem to remember some weird quirk if the `optional` is added when it's actually. I think I just omit this if it's required. If you haven't run into any issues with local testing or testing on GA, then it's fine

**Comment by jiavila** [comment]
@pvelasco I was reviewing it...

**Comment by pvelasco** [comment]
Sorry :sad:

**Pull request merged**